jenkins/automation/alex.groovy

```diff
@@ -6,13 +6,13 @@ pipeline {
       }
       parameters {
           string(
-              defaultValue: 'centos-7.9.2009',
+              defaultValue: 'ssc-vm-g4-rhev4-0465.colo.seagate.com',
```

Comment:
Let's not use and hardcoded VM node. Please try to use a container.

Reply:
Looks good but it still uses VM. Any issue in executing it in the container?

Reply:
Yes I am facing a lot of issues on rockylinux-8.4 and centos-7.9.2009. Tried to troubleshoot but it is failing to execute the python file and throwing weird errors.

Reply:
Ok. No problem. Let's take that as an improvement.